package.json

```diff
@@ -6,6 +6,7 @@
     ".next/cache"
   ],
   "scripts": {
+    "dev-https": "next dev --experimental-https",
```

Comment:
Could you remove the `https `script ?

Reply:
This was added to make it easier to run https in dev mode